test/Microsoft.Framework.Internal.Test/BufferedHtmlContentTest.cs

```diff
@@ -131,20 +150,22 @@ public void WriteTo_WritesAllItems()
 
             // Assert
             Assert.Equal(2, content.Entries.Count);
-            Assert.Equal("Written from TestHtmlContent: HelloTest", writer.ToString());
+            Assert.Equal("Written from TestHtmlContent: HelloHtmlEncode[[Test]]", writer.ToString());
         }
 
+        // We're purposely avoiding anything here that actually gets encoded. We don't want to take a dependency
+        // on the actual encoder implementation.
```

Comment:
This test is for the `ToString` method. I was about 50/50 before on just removing the ToString() and replacing it with a debugger display. Now I feel pretty strongly that removing ToString and this test is the right call. Thoughts?